src/js/lib/content/src/index.ts

```diff
@@ -566,21 +667,104 @@ function tryAsString(value: any): string {
     return value.toString()
 }
 
-/// Main entry point. Loads WASM, initializes it, chooses the scene to run.
-API.main = async function (inputConfig: any) {
-    const urlParams = new URLSearchParams(window.location.search)
-    // @ts-ignore
-    const urlConfig = Object.fromEntries(urlParams.entries())
+async function fetchApplicationConfig(config: Config) {
+    const https = require('https')
 
-    const config = Config.default()
-    config.updateFromObject(inputConfig)
-    config.updateFromObject(urlConfig)
+    return new Promise((resolve: any, reject: any) => {
+        https.get(config.application_config_url, (res: any) => {
+            if (res.statusCode !== 200) {
```

Comment:
200 is hardcoded literal. Create a well-named const of what this error means and usethe const instead